### ISSUE TITLE
Install: "Unable to find a suitable version for angular"

### ISSUE THREAD

**Issue by qamil** [KrekkieD/nocca#47]
Currently bower can't decide on an angular version during an install:
```
Unable to find a suitable version for angular, please choose one:
    1) angular#1.2.28 which resolved to 1.2.28 and is required by angular-mocks#1.2.28
    2) angular#~1.2.0 which resolved to 1.2.28 and is required by angular-truncate#997d4b4bda
    3) angular#>= 1.0.8 which resolved to 1.2.28 and is required by ui-router#0.2.15
    4) angular#~1.3.2 which resolved to 1.3.17 and is required by ng-websocket#0.2.1
    5) angular#1.3.17 which resolved to 1.3.17 and is required by angular-messages#1.3.17
    6) angular#^1.3.0 || >1.4.0-beta.0 which resolved to 1.3.17 and is required by angular-material#0.9.8
    7) angular#1.4.2 which resolved to 1.4.2 and is required by angular-animate#1.4.2, angular-aria#1.4.2
    8) angular#~1.4.0 which resolved to 1.4.2 and is required by nocca
```
As far as I can tell, putting this in your bower.json fixes this:
```
"resolutions": {
    "angular": "~1.4.0"
  }
```
But I'm not sure which version is the most appropriate.

Is there any other way around this?

**Comment by KrekkieD** [comment]
Should be good now :)

**Comment by KrekkieD** [comment]
Included in 0.4.2 release, published to NPM.

**Comment by qamil** [comment]
Awesome :+1: Will test it immediately
